### ISSUE TITLE
Enable only for specific file types

### ISSUE THREAD

**Issue by trsdln** [kamykn/spelunker.vim#4]
This plugin is great! Thanks!

There is one thing I was wondering about: Is is possible to enable spell check only for specific files or even better: enable it only when internal `set spell`  is enabled?

**Comment by kamykn** [comment]
Hi, @trsdln!

It's probably easier to add option to enable or disable this plugin by filetype.
How about switching enable this plugin by file type?

**Comment by trsdln** [comment]
@kamykn yep, probably it's easier. What say about this #5 approach ?

**Comment by trsdln** [comment]
doesn't break anything. but can be customized using:

```vim
let g:spelunker_disable_auto_group = 1
augroup spelunker
  autocmd!
  autocmd BufWinEnter,BufWritePost *.vim,*.js,*.jsx,*.json,*.md call spelunker#check()
augroup END
```

**Comment by kamykn** [comment]
Thank you for your pull request.
I merged #5 because it was a good feature.

Thank you!
